src/js-v2-api-specification.md

```diff
@@ -0,0 +1,309 @@
+---
+layout: base.njk
+title: Javascript API Specification
+---
+
+Any application embedded within the NHS App as part of a web integration has access to a limited number of methods to interact with the native Android or iOS Applications using an exposed Javascript interface.
+
+The NHS App JS API must be loaded inline rather than being bundled into the client application codebase. This is so that any changes to the implementation of the API do not require client applications to be recompiled and redeployed.
+
+To include the Javascript interface:
+
+```javascript
+<script type="text/javascript" src="<env-url>/js/v1/nhsapp.js"></script>
+```
+
+Where `env-url` will be determined by the environment being used. Unless testing un-released functionality, the NHS App production environment can be used `https://www.nhsapp.service.nhs.uk/`.
+
+Example Usage:
+
+```javascript
+nhsapp.navigation.goToHomepage()
+```
+
+## Reference
+* [Tools](/nhsapp-developer-documentation/js-api-specification#tools)
+  * [getAppPlatform](/nhsapp-developer-documentation/js-api-specification#getAppPlatform)
+  * [isOpenInNHSApp](/nhsapp-developer-documentation/js-api-specification#isOpenInNHSApp)
+* [Navigation](/nhsapp-developer-documentation/js-api-specification#navigation)
+  * [setBackAction](/nhsapp-developer-documentation/js-api-specification#setBackAction)
+  * [clearBackAction](/nhsapp-developer-documentation/js-api-specification#clearBackAction)
+  * [goToHomepage](/nhsapp-developer-documentation/js-api-specification#goToHomepage)
+  * [goToPage](/nhsapp-developer-documentation/js-api-specification#goToPage)
+  * [openBrowserOverlay](/nhsapp-developer-documentation/js-api-specification#openBrowserOverlay)
+  * [openExternalBrowser](/nhsapp-developer-documentation/js-api-specification#openExternalBrowser)
+* [Storage](/nhsapp-developer-documentation/js-api-specification#storage)
+  * [addToCalendar](/nhsapp-developer-documentation/js-api-specification#addToCalendar)
+  * [downloadFromBytes](/nhsapp-developer-documentation/js-api-specification#downloadFromBytes)
+
+### Tools <a name="tools"></a>
+
+#### getAppPlatform Function <a name="getAppPlatform"></a>
+
+`nhsapp.tools.getAppPlatform()`
+
+##### Description
+
+Returns a string indicating the platform on which the current page is being displayed. Check is based on the current User Agent.
+
+The returned string will be one of the following:
+
+- "android" - indicates that the page is being displayed within the native Android NHS App.
+- "ios" - indicates that the page is being displayed within the native iOS NHS App.
+- "none" - indicates that the page is not being displayed within the native NHS App (i.e. it is probably being displayed in a web browser).
+
+##### Status
```

Comment:
Are any of these Status sections valuable if they're all "Live"?

Reply:
This is existing text. Might be having some different purpose.That's reason I am not deleting.